src/Language/PureScript/CoreFn/ToJSON.hs

```diff
@@ -27,64 +30,68 @@ constructorTypeToJSON :: ConstructorType -> Value
 constructorTypeToJSON ProductType = toJSON "ProductType"
 constructorTypeToJSON SumType = toJSON "SumType"
 
+infixr 8 .=
+(.=) :: ToJSON a => String -> a -> Pair
```

Comment:
Almost everything in this module is due to `aeson-2` changes, specifically the left hand side of `.=` must now be a `Key` rather than a `Text`. I thought it might be easier to define our own version of `.=` so that we don't have to convert the key to a different type each time we call it.